tfrecorder/client.py

```diff
@@ -64,19 +64,66 @@ def _validate_runner(
         'DataflowRunner requires a tfrecorder whl file for remote execution.')
 
 
-# def read_image_directory(dirpath) -> pd.DataFrame:
-#   """Reads image data from a directory into a Pandas DataFrame."""
-#
-#   # TODO(cezequiel): Implement in phase 2.
-#   _ = dirpath
-#   raise NotImplementedError
+def _path_split(filepath: str) -> Tuple[str, str]:
+  """Splits `filepath` into (head, tail) where `tail` is the last pathname.
+
```

Comment:
This (head, tail) thing is confusing to me, but I see that's what's also used in os.path.split.  Strange wording, but at least we are consistent.

Reply:
Yeah `head` and `tail` aren't very specific. I added an example output of the function in the docstring.